basil/HL/weiss_labevent.py

```diff
@@ -22,12 +23,23 @@ class weissLabEvent(HardwareLayer):
         -8: "Data could not be read!",
     }
 
+    STATUS_CODES = {
+        1: 'Test not running (Idle)',
+        3: 'Test running',
+        4: 'Warnings present',
+        8: 'Alarms present'
+    }
+
     def __init__(self, intf, conf):
         super(weissLabEvent, self).__init__(intf, conf)
 
     def init(self):
         super(weissLabEvent, self).init()
 
+        info = self.get_info()
+        if info != 'LabEvent T/210/70/5':
+            logging.error('Not the expected climatechamber! Expected {0}, got {1}'.format('LabEvent T/210/70/5', info))
```

Comment:
In basil we define a logger per file at the beginning of the module
```python
import logging
logger = logging.getLogger(__name__)
```

Reply:
Otherwise we cannot filter logs efficiently in the DAQs that use basil. See [here](https://github.com/SiLab-Bonn/basil/pull/79)

Reply:
done